movement/watch_faces/complication/countdown_face.c

```diff
@@ -192,7 +195,7 @@ bool countdown_face_loop(movement_event_t event, movement_settings_t *settings,
             break;
         case EVENT_ALARM_LONG_PRESS:
             if (state->mode == cd_setting) {
-                    state->minutes = DEFAULT_MINUTES;
+                    state->minutes = 0;
```

Comment:
The problem is that it's very hard to get to less than DEFAULT_MINUTES. i.e. in my opinion the face is semi-broken in these cases, so I'd like to fix that.

I'm reluctant to make the default value configurable at runtime, as this would make the face quite complex. Note that it 'remembers' the last value you used, so there is some kind of default.

Reply:
As this is @tahnok's watch face, I'd defer to him on how he'd prefer to approach this.

Reply:
I think this is OK, although I'm curious to hear more about your usecase @wryun . Do you often change what the timer is? Is there a set of times you usually use (1, 3, 5, 25, 2hr)?

I ask because I think there's room to have some 'saved' countdowns without being overly complex

Reply:
To be honest, I haven't used this much yet, but I have used mobile phone countdowns in the past with my kids, and it's useful for me to have 1 minute countdowns. This is the main motivation for the PR, as the existing code (unless I'm missing something) made it very hard to have 1 minute countdowns.